P1/Project_Code/frontend/src/components/Layout/Layout.jsx

```diff
@@ -25,16 +26,10 @@ export const Layout = () => {
 
   return (
     <Grid container spacing={0}>
-      <Grid
-        item
-        xs={1}
-        container
-        justifyContent="center"
-        style={{ paddingTop: "1rem" }}
-      >
+      <NavigationBarGridContainer item xs={0} md={0} lg={1} container justifyContent="center">
         <NavigationBar></NavigationBar>
-      </Grid>
-      <Grid item xs={11}>
+      </NavigationBarGridContainer>
+      <Grid item xs={12} md={12} lg={11}>
```

Comment:
you should be able to remove the md={12}.  MaterialUI adopts/inherits the value defined at a lower screen size (xs={12})

Reply:
Agreed, this code should be simplified.